src/stream/src/executor/now.rs

```diff
@@ -99,24 +99,27 @@ impl<S: StateStore> NowExecutor<S> {
                 let time_millis = Epoch::from(barrier.epoch.curr).as_unix_millis();
                 let timestamp = Some(ScalarImpl::Int64((time_millis * 1000) as i64));
 
-                let data_chunk = DataChunk::from_rows(
-                    &if last_timestamp.is_some() {
```

Comment:
what if `last_timestamp` == `None` ?

Reply:
I just split the code upon last_timestamp.is_some() branch, it's easier to read IMO.